pkg/mig/mode/nvml_test.go

```diff
@@ -52,38 +52,38 @@ func NewMockNvmlLunaServer() *nvmlMigModeManager {
 
 func (d *mockNvmlA100Device) SetMigMode(mode int) (Return, Return) {
```

Comment:
At which point do we replace the usages of `mockNvmlA100Device` with those from the `mock` package? (or implement them there)?

Reply:
What do you mean by "at which point"? At which point in the current code? They are not replacements for the ones in the `mock` package, but rather extend from them to provide logic necessary for testing the abstractions in mig-parted specifically.

Reply:
This was unclear from my side. The question should have been whether it makes sense to move these abstractions to a common location. And if it makes sense, does it have to be done as part of this PR. I think your answer makes it clear that these are not as common as I thought on the first read-through.